docs/user/urls.md

```diff
@@ -0,0 +1,117 @@
+---
+sidebar_position: 10
+---
+
+# Agama URLs
+
+When you want to perform an unattended installation, Agama needs to know where
+to find the profile. Usually, you make use of the `inst.auto` argument, which
+receives a URL. Agama supports a wide range of protocols, including
+`http(s)://`. `ftp://`, `file://`, etc. However, it supports most of the
+[AutoYaST specific URLs](https://doc.opensuse.org/projects/autoyast/#Commandline-ay)
+too.
+
+## Supported schemes
+
+Agama supports the following standard schemes[^1]:
+
+- `http` and `https` for HTTP and HTTPS.
+- `ftp` and `ftps` for FTP and FTP over TSL.
+- `smb` and `smbs` for the Server Message Block (SMB) protocol[^2].
+- `nfs` for the Network File System (NFS) protocol.
+- `file` for local files.
+
+In addition to those schemes, Agama supports a set of specific ones which can be
+rather useful:
+
+- [`device`](#device) for finding files in specific devices (even unmounted
```

Comment:
There is an interesting discussion whether something is "in the device" or "on the device" https://english.stackexchange.com/questions/326475/on-hard-drive-versus-in-memory

My brain still sees that "on the device", although SSDs don't have any platters anymore and thus it would logically be "in the device" instead, but historically it's "on". This is a computer-speak, "in" is a correct English way :D

Reply:
I will check with the documentation team.

Reply:
I am following up on your suggestion after checking with the docu team, although both options are correct. Actually, we have an interesting thread in Slack talking about that :wink: